### ISSUE TITLE
[Accessibility] Make the Assassin card (black card) darker to allow easier differentiation between already guessed cards

### ISSUE THREAD

**Issue by stechedo** [phantomesse/code-names#44]
In Spymaster view, the dark grey of the Assassin card is very similar to the colors that represent already guessed cards.  

Making the Assassin card darker (i.e. solid black with white text) would make it easier to differentiate this card.
![Screen Shot 2020-12-12 at 11 40 11 AM](https://user-images.githubusercontent.com/42300227/101989556-2b465c80-3c6f-11eb-8ce9-f4807bf2025f.png)
